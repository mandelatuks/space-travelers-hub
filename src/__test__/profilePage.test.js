import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../Redux/Configurestore';
import Profile from '../components/pages/ProfilePage';

test('test the MissionPage snapshot', () => {
  const missionRenderer = renderer.create(
    <Provider store={store}>
      <BrowserRouter>

        <Profile />
      </BrowserRouter>
    </Provider>
    ,
  ).toJSON();
  expect(missionRenderer).toMatchSnapshot();
});
