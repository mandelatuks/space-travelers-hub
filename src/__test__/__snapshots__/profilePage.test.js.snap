// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the MissionPage snapshot 1`] = `
<section
  className="profile-page"
>
  <div>
    <h1>
       My Missions
    </h1>
    <ul />
  </div>
  <div>
    <h1>
      My Rockets
    </h1>
    <ul />
  </div>
</section>
`;
